plugin.video.fsgo/default.py

```diff
@@ -0,0 +1,376 @@
+﻿# -*- coding: utf-8 -*-
+"""
+A Kodi plugin for FOX Sports GO
+"""
+import sys
+import os
+import urllib
+import urlparse
+from datetime import datetime
+
+from resources.lib.fsgo import fsgolib
+
+import xbmc
+import xbmcaddon
+import xbmcvfs
+import xbmcgui
+import xbmcplugin
+
+addon = xbmcaddon.Addon()
+addon_path = xbmc.translatePath(addon.getAddonInfo('path'))
+addon_profile = xbmc.translatePath(addon.getAddonInfo('profile'))
+language = addon.getLocalizedString
+logging_prefix = '[%s-%s]' % (addon.getAddonInfo('id'), addon.getAddonInfo('version'))
+
+if not xbmcvfs.exists(addon_profile):
+    xbmcvfs.mkdir(addon_profile)
+
+_url = sys.argv[0]  # get the plugin url in plugin:// notation
+_handle = int(sys.argv[1])  # get the plugin handle as an integer number
+
+cookie_file = os.path.join(addon_profile, 'cookie_file')
+credentials_file = os.path.join(addon_profile, 'credentials')
+
+if addon.getSetting('debug') == 'false':
+    debug = False
+else:
+    debug = True
+
+if addon.getSetting('verify_ssl') == 'false':
+    verify_ssl = False
+else:
+    verify_ssl = True
+
+fsgo = fsgolib(cookie_file, credentials_file, debug, verify_ssl)
+
+
+def addon_log(string):
+    if debug:
+        xbmc.log('%s: %s' % (logging_prefix, string))
```

Comment:
Please set the logging level to xbmc.LOGDEBUG as the default for Jarvis is still xbmc.LOGNOTICE (that will change for krypton and future versions). The rational is not to have the log file too spammed and to have a centralized way of enabling debugging